### PR TITLE
Mangaworld changed domain

### LINKED ISSUE #9291: Mangaworld: Changed domain
### Source information

mangaworld.in

### Steps to reproduce

To change the domain from https://www.mangaworld.io to https://www.mangaworld.in

### Expected behavior

To work

### Actual behavior

Doesn't work

### Tachiyomi version

Last

### Android version

11

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by ildallo**
.io to .in

closes #9291

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Annotated `Source` or `SourceFactory` classes with `@Nsfw` when appropriate
- [x] Added the `containsNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by alessandrojean** [comment]
Follow the checklist:

> Updated `extVersionCode` value in build.gradle for individual extensions

**Comment by alessandrojean** [comment]
Did you compiled the extension and tested it?

**Review by SmashBanana — APPROVED** [review]

**Comment by ildallo** [comment]
> Did you compiled the extension and tested it through Android Studio?
Yes

File: src/it/mangaworld/src/eu/kanade/tachiyomi/extension/it/mangaworld/Mangaworld.kt [hunk 1]
```diff
@@ -150,16 +150,16 @@ class Mangaworld : ParsedHttpSource() {
         val metaData = document.select("div.comic-info").first()
 
         val manga = SManga.create()
-        manga.author = infoElement.select("a[href^=https://www.mangaworld.io/archive?author=]").first()?.text()
-        manga.artist = infoElement.select("a[href^=https://www.mangaworld.io/archive?artist=]")?.text()
+        manga.author = infoElement.select("a[href^=https://www.mangaworld.in/archive?author=]").first()?.text()
```
Comment (arkon) [inline src/it/mangaworld/src/eu/kanade/tachiyomi/extension/it/mangaworld/Mangaworld.kt:153]: Can you update these to refer to `baseUrl` instead?

**Review by arkon — CHANGES_REQUESTED** [review]

**Comment by Gyro7** [comment]
Thanks god, I was going to work on it but you already did it!

File: src/it/mangaworld/src/eu/kanade/tachiyomi/extension/it/mangaworld/Mangaworld.kt [hunk 2]
```diff
@@ -150,16 +150,16 @@ class Mangaworld : ParsedHttpSource() {
...
+        manga.artist = infoElement.select("a[href^=https://www.mangaworld.in/archive?artist=]")?.text()
```
Comment (Riztard) [inline src/it/mangaworld/src/eu/kanade/tachiyomi/extension/it/mangaworld/Mangaworld.kt:154]: this too and the other better to refer to `baseurl` as arkon said

**Comment by ildallo** [comment]
the code works perfectly, just approve it

File: src/it/mangaworld/src/eu/kanade/tachiyomi/extension/it/mangaworld/Mangaworld.kt (hunk 1 above)
Comment (arkon) [reply src/it/mangaworld/src/eu/kanade/tachiyomi/extension/it/mangaworld/Mangaworld.kt:153]: You didn't actually resolve this.

**Comment by CarlosEsco** [comment]
> 
> 
> the code works perfectly, just approve it

fix the requested changes

**Comment by Gyro7** [comment]
So? Why did you close it?

**Comment by CarlosEsco** [comment]
he obviously didn't want to actually test his code, or make the suggested changes

**Pull request closed without merging**